Java/libraries/recognizers-text-date-time/src/main/java/com/microsoft/recognizers/text/datetime/parsers/BaseDurationParser.java

```diff
@@ -55,11 +55,11 @@ public DateTimeParseResult parse(ExtractResult er, LocalDateTime reference) {
 
             if (innerResult.getSuccess()) {
                 innerResult.setFutureResolution(ImmutableMap.<String, String>builder()
-                        .put(TimeTypeConstants.DURATION, StringUtility.format((Double)innerResult.getFutureValue()))
+                        .put(TimeTypeConstants.DURATION, StringUtility.format((Double) innerResult.getFutureValue()))
```

Comment:
Nit: Can we tweak the current linting rules? We typically don't add spaces after casts.